extensions.json

```
@@ -219,6 +219,10 @@
       "id": "denoland.vscode-deno",
       "repository": "https://github.com/denoland/vscode_deno"
     },
+    {
+      "id": "DigitalBrainstem.javascript-ejs-support",
+      "repository": "https://github.com/Digitalbrainstem/ejs-grammar"
```

I'd suggest to specify a version to ensure we publish exactly from the sources of that release:

```suggestion
      "repository": "https://github.com/Digitalbrainstem/ejs-grammar",
      "version": "1.2.1",
      "checkout": "v1.2.1"
```